backend/routers/productRouter.js

```diff
@@ -36,7 +38,13 @@ productRouter.get(
         : order === 'toprated'
         ? { rating: -1 }
         : { _id: -1 };
-
+    const count = await Product.count({
```

Comment:
DeprecationWarning: collection.count is deprecated, and will be removed in a future version. Use Collection.countDocuments or Collection.estimatedDocumentCount instead